packages/frontend/model/advertisement.ts

```diff
@@ -1,5 +1,25 @@
+import { AdSlotParameters } from '@frontend//web/components/AdSlot';
+
 // We are using this function to control the activation of the commercial features
 // Currently it reports that the user has opted in to a 0% AB test.
 export const shouldDisplayAdvertisements = (config: ConfigType) => {
     return config.abTests.dotcomRenderingAdvertisementsVariant === 'variant';
 };
+
+export const adSlotParameters = () => {
```

Comment:
Why are you exporting a function rather than just an object that you can then pass straight into https://github.com/guardian/dotcom-rendering/pull/739/files#diff-4b2a851e84aa284d7eede03be3a3de8dR64 ?

Will there be dynamic functionality in the future here?

Reply:
> Will there be dynamic functionality in the future here?

Yes dynamic functionality, I am testing with hard coded values but that function will start sending the correct . Sorry I could/should have mentioned this in the PR.